getting-started/main.cpp

```diff
@@ -139,8 +139,14 @@ static void sign_a_message_using_rsa(const uint8_t *key, size_t key_len)
 {
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    uint8_t hash[] = "INPUT_FOR_SIGN";
-    uint8_t signature[PSA_ASYMMETRIC_SIGNATURE_MAX_SIZE] = {0};
+    psa_algorithm_t alg = PSA_ALG_RSA_PKCS1V15_SIGN(PSA_ALG_SHA_256);
+    // SHA-256 of {'a', 'b', 'c'}
+    static const uint8_t hash[PSA_HASH_SIZE(PSA_ALG_SHA_256)] = {
+        0xba, 0x78, 0x16, 0xbf, 0x8f, 0x01, 0xcf, 0xea, 0x41, 0x41, 0x40, 0xde,
+        0x5d, 0xae, 0x22, 0x23, 0xb0, 0x03, 0x61, 0xa3, 0x96, 0x17, 0x7a, 0x9c,
+        0xb4, 0x10, 0xff, 0x61, 0xf2, 0x00, 0x15, 0xad
+    };
+    uint8_t signature[PSA_ASYMMETRIC_SIGNATURE_MAX_SIZE];
```

Comment:
Looks like the `= {0};` got dropped off here. This means there may be extra garbage at the end of the buffer after the sign() call, rather than clean 0 when you hexdump the signature buffer.

Reply:
Done